.github/workflows/code-quality.yml

```
@@ -0,0 +1,196 @@
+          fail_ci_if_error: false
+      
+      - name: Laravel specific checks
+        run: |
+          # Check for env() usage outside config files
+          echo "Checking for env() usage outside config files..."
+          ! grep -r "env(" app/ --include="*.php" || echo "Warning: env() found in app files"
+          
+          # Check for dd() or dump() in code
+          echo "Checking for debug statements..."
+          ! grep -r "dd(\|dump(" app/ config/ --include="*.php" || echo "Warning: Debug statements found"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)